docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2153,6 +2160,8 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/persistence/common/updateCreateEnabled`|true|
 |`fhirServer/persistence/datasources`|embedded Derby database: derby/fhirDB|
 |`fhirServer/persistence/datasources/<datasourceId>/type`|derby|
+|`fhirServer/persistence/datasources/<datasourceId>/jndiName`|`jndi/fhir_[tenantId]_[datastoreId]`|
```

Comment:
sorry, I just noticed that we call it `datasouorceId` in the property name and it really does need to match, so I'm ammending my proposal:
```suggestion
|`fhirServer/persistence/datasources/<datasourceId>/jndiName`|`jndi/fhir_[tenantId]_[datasourceId]`|
```

Reply:
honestly, I sort of wish we called them dataStores instead of dataSources (had a convo with Robin about this), but easier just to change the value in the pattern to match :-)

Reply:
one reason for preferring `datastoreId` is that it makes it more clear that this is distinct from the server.xml `id` attribute:
```
<dataSource id=""
```

one reason for preferring `datasourceId` is that the fhir-server-config property group is called "datasources" and so logically under there the ids would be "datasourceIds".

Reply:
same here